arduino-core/src/processing/app/BaseNoGui.java

```diff
@@ -979,6 +977,13 @@ static public void selectBoard(TargetBoard targetBoard) {
     File platformFolder = targetPlatform.getFolder();
     PreferencesData.set("runtime.platform.path", platformFolder.getAbsolutePath());
     PreferencesData.set("runtime.hardware.path", platformFolder.getParentFile().getAbsolutePath());
+
+    if (!recentlyUsedBoards.contains(targetBoard)) {
+      recentlyUsedBoards.add(targetBoard);
+    }
+    if (recentlyUsedBoards.size() > 4) {
```

Comment:
Should the size of the LRU list be a preference? I'm pretty sure people will be asking for more entries soon :-)

Reply:
Agree, a preference will not hurt and will be future proof :slightly_smiling_face: